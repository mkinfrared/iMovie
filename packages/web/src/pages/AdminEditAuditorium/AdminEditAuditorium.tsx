import React from "react";
import { useHistory, useParams } from "react-router-dom";

import EditAuditoriumForm from "containers/EditAuditoriumForm";

type RouteParams = {
  cinemaId: string;
  auditoriumId: string;
};

const AdminEditAuditorium = () => {
  const { cinemaId, auditoriumId } = useParams<RouteParams>();
  const { goBack } = useHistory();

  return (
    <EditAuditoriumForm
      cinemaId={parseInt(cinemaId, 0)}
      auditoriumId={parseInt(auditoriumId, 0)}
      onCancel={goBack}
    />
  );
};

export default React.memo(AdminEditAuditorium);
