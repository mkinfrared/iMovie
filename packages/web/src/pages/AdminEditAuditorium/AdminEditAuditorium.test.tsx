import { render } from "@testing-library/react";
import React, { ReactElement } from "react";

import AdminEditAuditorium from "./AdminEditAuditorium";

describe("<AdminEditAuditorium />", () => {
  let Component: ReactElement;

  beforeEach(() => {
    Component = <AdminEditAuditorium />;
  });

  it("should be defined", () => {
    const { container } = render(Component);

    expect(container).toBeDefined();
  });

  it("should match a snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });
});
