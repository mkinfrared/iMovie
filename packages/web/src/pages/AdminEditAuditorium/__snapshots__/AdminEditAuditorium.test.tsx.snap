// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminEditAuditorium /> should match a snapshot 1`] = `
<div>
  <div>
    AdminEditAuditorium
  </div>
</div>
`;
